examples/grpc-server/message.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package llama;
+
+option go_package = "./pkg/grpc";
+
+service LlamaGoService {
+    rpc Answer(Job) returns (stream Output){}
+    rpc Embed(Job) returns (Output){}

```

Comment:
using the same output type doesn't make a ton of since IMO, I would separate this out as: `rpc CreateEmbedding(CreateEmbeddingRequest) returns (Embedding) {}`

Reply:
right